README.md

```diff
@@ -60,6 +60,26 @@ If someone already merged it but didn't release it, then it can be released:
 $ ../../bin/release-module
 ```
 
+### Stable releases
+
+All the release scripts assume they're releasing to master. For a stable release, additional parameters are needed. All of the above scripts accept parameters. The first is the directory where to execute. To create a release PR for the current directory to the 11.1-stable branch:
+
+```console
+$ ../../bin/release-pr . 11.1-stable
```

Comment:
This assumes you have this structure, correct?

```
foreman-installer-module-sync
├── modules
│   └── puppet-foreman_proxy_content
```

That's not clear from the README. Is there a script or command that sets this up?

Reply:
I ran this, and this was the output (with no changes):

```
ehelms@war puppet-foreman_proxy_content (master)$ ../../bin/release-pr . 11.1-stable
Switched to a new branch 'release-12.0.0'
On branch release-12.0.0
nothing to commit, working tree clean
```

Reply:
> This assumes you have this structure, correct?

Well yes, but it already did that before. Just keeping things consistent. Technically you can also run it from the root as `./bin/release-pr modules/puppet-foreman_proxy_content 11.1-stable` but I always like to work in the directory so you have tools like `git diff`.

> I ran this, and this was the output (with no changes):

It is intended that you run this instead of the others. So what I always did was `git checkout 11.1-stable`, build up a changelog (often manually because it was easier than using the tool), then ran this. You may also need to run `bundle install` or `bundle update`.

Reply:
What does build up a changelog mean? Manually adding the entries in the CHANGELOG file?

Reply:
I've rewritten it a bit. Please have a look at the latest version.